src/pages/Main/MainPage.tsx

```diff
@@ -1,28 +1,17 @@
-// /* eslint-disable */
-// // @ts-nocheck
-import { useEffect } from 'react';
-import { NavLink } from 'react-router-dom';
+import React, { useEffect } from 'react';
 import styles from './MainPage.module.scss';
 import { useAppDispatch, useAppSelector } from '../../redux/store';
 import { getDataMainPage } from '../../redux/reducers/pagesDataReducer';
 import Header from '../../components/Header/Header';
 import { Footer } from '../../components/Footer/Footer';
+import Banner from './Banner/Banner';
+import Manual from './Manual/Manual';
+import TestMenu from './TestMenu/TestMenu';
+import LastSection from './LastSection/LastSection';
+import Direction from './Direction/Direction';
 
-import icon001 from '../../assets/icons/icon_001.png';
-import icon002 from '../../assets/icons/icon_002.png';
-import icon003 from '../../assets/icons/icon_003.png';
-import icon004 from '../../assets/icons/icon_004.png';
-import icon005 from '../../assets/icons/icon_005.png';
-import iconManual001 from '../../assets/icons/icon_manual_001.png';
-import iconManual002 from '../../assets/icons/icon_manual_002.png';
-import iconManual003 from '../../assets/icons/icon_manual_003.png';
-import iconManual004 from '../../assets/icons/icon_manual_004.png';
-// import testMenu from '../../assets/img/test_menu.png';
-import CommonButton from '../../UI/CommonButton/CommonButton';
-import { Path } from '../constants/path';
-import { Slider } from './Slider/Slider';
-
-export default function MainPage() {
+// eslint-disable-next-line
+const MainPage: React.FC = () => {
```

Comment:
I think we need use 1 type of name components
Do you have any reason for this sintaxis?

Reply:
i agree, fix this